### DISCUSSION TITLE
OCIO v2

### DISCUSSION THREAD

**Discussion by BrianHanke** [AcademySoftwareFoundation/OpenRV#77] [General]
Has there been any research/work on upgrading OpenRV to support OCIO v2?

**Comment by compaga** [comment]
Hi BrianHanke, we already started the work to update to OCIOv2. Since OCIOv2 and backward compatible, first step is to simply update it. Revamping Open RV Colour management to support OCIOv2 properly would be a much bigger task we haven't started yet.

**Reply by chrisbrejon** [reply]
Hello, do you plan on releasing this OCIOv2 update soon ? Thanks !

**Reply by geffrak** [reply]
The work is actively in progress, but we can't share any date yet because it's a big change.

**Comment by bernie-laberge** [comment]
Closing this discussion since OCIOv2 support was merged some time ago in Open RV
